frontend/src/assets/css/fonts.scss

```diff
@@ -0,0 +1,23 @@
+@font-face {
+  font-weight: 400;
+  font-family: "Plus Jakarta Sans";
+  src: url("fonts/plusjakartasans-regular.woff2");
```

Comment:
are you sure it's working? file in public/fonts was not renamed to lowercase

Reply:
I had problems with git push, I had to roll back the branch and the file names remained the same. I'll fix it again now.